README.md

```diff
@@ -95,3 +95,33 @@ Other image props except `resizeMode` are accepted.
 ## Contributors
```

Comment:
Please help remove the old `Contributors` section.

Reply:
updated